packages/svelte/src/lib/context-provider.ts

```diff
@@ -0,0 +1,20 @@
+import { KeyedChatStore, setChatContext } from "./chat-context.svelte.js";
+import {
+  KeyedCompletionStore,
+  setCompletionContext,
+} from "./completion-context.svelte.js";
+import {
+  KeyedStructuredObjectStore,
+  setStructuredObjectContext,
+} from "./structured-object-context.svelte.js";
+
+export function createAIContext() {
```

Comment:
I went with a function rather than a component because it's typically more ergonomic -- you just call it in the `script` block of your layout and it doesn't add another layer of nesting to your HTML.